### ISSUE TITLE
deasync in worker_threads

### ISSUE THREAD

**Issue by icodeforlove** [abbr/deasync#121]
I cant seem to get this module working in [worker_threads](https://nodejs.org/api/worker_threads.html) (node v12.9.1).

I get all kinds of gnarly errors.

 - `FATAL ERROR: HandleScope::HandleScope Entering the V8 API without proper locking in place`
- `segmentation fault`
- `malloc: *** error for object 0x102a315e0: pointer being freed was not allocated`

**reproducible code**

```javascript
const {
	Worker, isMainThread
} = require('worker_threads');

if (isMainThread) {
	console.log('master:started');

	new Worker(__filename);
} else {
	console.log('slave:started');

	const deasync = require('deasync');
	deasync.sleep(100);
	console.log('works');
}
```

**Comment by icodeforlove** [comment]
Something like this does work inside of a worker_thread

```javascript
require('child_process').execSync('sleep 0.1');
```

**Comment by christian-bromann** [comment]
Please see this comment https://github.com/webdriverio/webdriverio/pull/4886#issuecomment-565824408:

```
-   uv_run(uv_default_loop(), UV_RUN_ONCE);
+   uv_run(GetCurrentEventLoop(v8::Isolate::GetCurrent()), UV_RUN_ONCE);
```

Applying this change to deasync.cc should fix deasync in worker_threads.

**Comment by abbr** [comment]
Fixed with deasync@0.1.17.
